v2/grafeas/client.go

```diff
@@ -137,6 +137,7 @@ func (g *Client) GetBuildDetail(ctx context.Context, ref reference.Canonical) (r
 		return repository.BuildDetail{}, err
 	}
 
+	// TODO: Figure out if we need to add a fallback project for build details.
```

Comment:
a good question still, should we pull this out and make it a separate issue?

Reply:
👍 we can keep this in the code as this is a common practice. I will link the issue there once I create it